webapi/src/test/scala/org/knora/webapi/e2e/admin/GroupsADME2ESpec.scala

```diff
@@ -74,6 +76,60 @@ class GroupsADME2ESpec extends E2ESpec(GroupsADME2ESpec.config) with GroupsADMJs
             }
         }
 
+        "given a custom Iri" should {
```

Comment:
@subotic here are the tests for the optional `id`

Reply:
@subotic The optional `id` of the `user` is tested in the same way, please see [here](https://github.com/dasch-swiss/knora-api/pull/1668/files#r450019067)